templates/user/account_delete_form.html

```diff
@@ -0,0 +1,70 @@
+{% extends "base.html" %}
+
+{% block title %}My user profile{% endblock title %}
+
+{% block content %}
+<!-- Begin Page Content -->
+<div class="container-sm py-5">
+    <!-- Main Content Here -->
+    <div class="card border-0 shadow p-4">
+
+        <form action="{% url 'delete_account_post_handler' user.id %}" method="POST">
+            {% csrf_token %}
+            <div class="card-body">
+                <div class="row">
+                    <div class="col">
+                        <h5 class="card-title mb-0">Delete Account</h5>
+
+                        <div class="row pt-4">
+                            <p class="alert alert-warning" role="alert">
+                                When you delete your account, you lose access to SciLifeLab Serve account services.
+                                To delete you account, you must no longer own any projects or apps in Serve.
+                            </p>
+                        </div>
+
+                        <div class="row pt-4">
+                            {% if account_can_be_deleted == True %}
+                            <p class="alert alert-info" role="alert">
+                            You do not own any project in Serve and this user account can be deleted.
+                            </p>
+                            {% else %}
+                            <p class="alert alert-danger" role="alert">
+                            This user user account cannot be deleted because you are the owner of at least one Serve project.
+                            Please delete or transfer ownership to another user account.
+                            </p>
+                            {% endif %}
+                        </div>
+
+                        <div class="row py-4">
+                            <div class="form-check">
+                                <input class="form-check-input" type="checkbox" value="" id="checkDeleteAccount"
+                                {% if account_can_be_deleted == False %}disabled{% endif %} >
+
+                                <label class="form-check-label" for="checkDeleteAccount">
+                                I have understood the consequences of deleting my user account.
+                                </label>
+                              </div>
+                        </div>
+
+                    </div>
+                </div>
+          </div>
+
+          <div class="card-footer d-flex justify-content-between">
+            <a class="btn btn-outline-secondary" href="{% url 'user-profile' %}">Cancel</a>
+            <button type="submit" class="btn btn-danger" id="buttonDoDeleteAccount" disabled>Warning! Delete your account</button>
```

Comment:
We have the opposite order on all other cards, so maybe keep that.
```suggestion
            <button type="submit" class="btn btn-danger" id="buttonDoDeleteAccount" disabled>Delete</button>
            <a class="btn btn-primary" href="{% url 'user-profile' %}">Cancel</a>
```

Reply:
On all other forms we have Cancel on the left and the action button on the right, like this. However I chose to have the action button in red (danger) because this signals that the delete action will result in irrevocable consequences. I think that we should change the color of all of the Cancel buttons to another color that is not red.

Reply:
Interesting. Except for when opening - deleting a project, then the delete is on the left in red and open/close project on the right. We should review all buttons and make them consistent.

Reply:
Yes, this is what i though aswell.